packages/asset-swapper/contracts/src/WooPPSampler.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+import "./interfaces/IWooPP.sol";
```

Comment:
nit/optional: I think we can inline interface in the same file since it's only used in this file.

Reply:
bumping this as well, its so small we can just define it in the same file